helmfile.d/README.md

```diff
@@ -55,6 +55,13 @@ Prefer to use this construct within state and stack files to ensure they can bui
 >
 > Set `disableValidationOnInstall` for releases that install custom resources that is not predefined in the environment.
 
+### Where to put manifests
+
+Releases may sometimes not be the best method to apply resources, especially for a create-once-update-never lifecycle.
```

Comment:
Nit:
```suggestion
Helm releases may sometimes not be the best method to apply resources, especially for a create-once-update-never lifecycle.
```

Reply:
Do you want that everywhere or just in the beginning so it is clear what kind of releases we're talking about?